org.eclipse.winery.repository.ui/src/app/instance/serviceTemplates/threatAssessment/threatAssessment.component.ts

```diff
@@ -0,0 +1,122 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache Software License 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+import { Component, OnInit, Input } from '@angular/core';
+import { ThreatAssessmentService } from './threatAssessment.service';
+
+export interface QName {
```

Comment:
Isn't there already a QName interface?

Reply:
only QNameWithTypeAPIData. Refactored to new class...?